### PR TITLE
KSYNC

### PR DESCRIPTION
**Pull request by troykessler**
This PR includes a doc section explaining how Cronos users can use KYVE's KSYNC in order to sync their nodes from genesis or how to perform historical state-syncs.

### CODE REVIEW

**Review by kaz-CDC — COMMENTED** [review]
Hi @troykessler have these changes been checked and verified?

**Review by kaz-CDC — COMMENTED** [review]
Hi @troykessler 
Do you want us to add the KYVE Network as a new sync tool to our documentation?

**Comment by troykessler** [comment]
> Hi @troykessler Do you want us to add the KYVE Network as a new sync tool to our documentation?

Yes, our sync tool KSYNC enables Cronos Validators to sync their nodes with the data archived permanently to Arweave by KYVE.

**Comment by kaz-CDC** [comment]
> > Hi @troykessler Do you want us to add the KYVE Network as a new sync tool to our documentation?
> 
> Yes, our sync tool KSYNC enables Cronos Validators to sync their nodes with the data archived permanently to Arweave by KYVE.

Got it! I read your documentation this weekend, I haven't Tested it yet. Have you submitted PR for this repo in the past?

File: for-node-hosts/running-nodes/cronos-mainnet/ksync.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# KSYNC
+
+This sections covers how to perform a genesis-sync up to live height and how to state-sync to any historical height for Cronos Mainnet with KSYNC. In summary KSYNC is a tool developed by [KYVE](https://www.kyve.network/) which is capable of syncing blocks and state-sync snapshots from the decentralized KYVE data lake directly into Cosmos blockchain nodes. In the case of Cronos all blocks and state-sync snapshots in a 10,000 interval have been validated in a decentralized way and permantenly archived to [Arweave](https://www.arweave.org/), a decentralized permanent storage solution. You can find full documentation on the tool [here](https://docs.kyve.network/validators/ksync).
+
+#### Installation
+
+You can install KSYNC with the following command, ensure that you have go1.21 installed:
+
+```bash
+go install github.com/KYVENetwork/ksync/cmd/ksync@latest
```
Comment (kaz-CDC) [inline for-node-hosts/running-nodes/cronos-mainnet/ksync.md:10]: Hi @troykessler 
1. Installed go: go version go1.22.5 darwin/amd64
2. mkdir - KSYNC
3. ran (zsh:) - KSYNC % go install github.com/KYVENetwork/ksync/cmd/ksync@latest (see image attached)
4. ran ksync version - zsh: command not found: ksync

Repeated the above steps in BASH (see image attached)
Screenshot 2024-07-21 at 18 35 53

Screenshot 2024-07-21 at 18 21 00

**Review by kaz-CDC — COMMENTED** [review]
Hi @troykessler please see my Comments.

**Comment by troykessler** [comment]
> Hi @troykessler please see my Comments.

Hey, sorry where can I view your comments? Can't find them

**Comment by kaz-CDC** [comment]
> > Hi @troykessler please see my Comments.
> 
> Hey, sorry where can I view your comments? Can't find them

Sorry Troy, it was a "typo". Please ignore.

**Comment by aw126** [comment]
Thank you for the PR @troykessler!

**Pull request merged**